projects/storefrontlib/shared/components/list-navigation/pagination/pagination.component.ts

```diff
@@ -61,6 +61,21 @@ export class PaginationComponent {
     );
   }
 
+  /**
+   * Format aria-label based on pagination page type.
+   *
+   * @param label
+   * @param item
+   * @returns
+   */
+  getAriaLabel(label: string, type: PaginationItemType) {
+    if (type === PaginationItemType.PAGE) {
+      return `${type} ${label}`;
+    }
+
+    return `${type} ${PaginationItemType.PAGE}`;
```

Comment:
Not necessary but if you wanted to shorten this code

```suggestion
    return type === PaginationItemType.PAGE
      ? `${type} ${label}`
      : `${type} ${PaginationItemType.PAGE}`;
```

Reply:
Changed